ui/src/lib.rs

```diff
@@ -1312,9 +1312,11 @@ enum Interrupt {
 impl Interrupt {
     fn char_press(&self) -> Option<char> {
         if let Self::KeyPressed(event) = &self {
-            if let KeyCode::Char(c) = event.code {
-                if c.is_alphanumeric() || c == '\'' {
-                    return Some(c)
+            if matches!(event.kind, KeyEventKind::Press) {
```

Comment:
On what occasions will this happen? This has never been reported before 😅 do you have any way to test this?

Reply:
This happens on windows, it happened to me and another searcher who reported it in flashbots discord the other day.
Works fine on linux though, dunno why